stories/SQFormDialogStepper.stories.js

```diff
@@ -25,77 +30,175 @@ const handleSubmit = (values, actions) => {
+          alert(`Create new account`);
+        }
+      },
+      ...mappedList
+    ];
+  };
+
+  const tabOptions = [
+    {
+      label: 'Agent PV',
+      value: 'agentPV',
+      listItems: quotedList(),
+      isSelectable: true
+    }
+  ];
+
+  const initialValues = {
+    friends: ['Joe', 'Jane', 'Jack', 'Jill'],
+    selectAll: false,
+    quote: ''
+  };
+  const names = [
+    'Jim',
+    'Jake',
+    'John',
+    'Jose',
+    'Jaipal',
+    'Joe',
+    'Jane',
+    'Jack',
+    'Jill'
+  ];
+
   return (
     <>
       <h3>Toggle the isOpen checkbox in the Knobs tab to view the Stepper</h3>
       <SQFormDialogStepper
-        title="SQ Stepper Form"
+        title="Quote Tool"
         isOpen={boolean('isOpen', false, 'Open/Close Dialog')}
         onClose={action('Close button clicked')}
-        contentHeight="25rem"
-        contentStyle={{height: '30rem'}}
+        contentStyle={{
+          height: '265px',
+          padding: '15px 15px'
+        }}
         fullWidth
         muiGridProps={{
           justify: 'space-between',
           alignItems: 'center'
         }}
-        initialValues={{
-          firstName: '',
-          lastName: '',
-          newAccount: false,
-          accountID: '',
-          description: '',
-          age: ''
-        }}
+        initialValues={initialValues}
         setValues={() => {
           console.log('values set');
         }}
         onSubmit={handleSubmit}
       >
-        <SQFormDialogStep
-          label="Personal Data"
-          validationSchema={object({
-            firstName: string().required('Required'),
-            lastName: string().required('Required name.')
-          })}
-        >
-          <SQFormTextField fullWidth name="firstName" label="First Name" />
-          <SQFormTextField fullWidth name="lastName" label="Last Name" />
-          <SQFormTextField fullWidth name="middleI" label="Middle Initial" />
-          <SQFormTextField fullWidth name="nickname" label="Nick-name" />
-          <SQFormTextField fullWidth name="alias" label="Alias" />
-          <SQFormCheckbox
-            name="newAccount"
-            type="checkbox"
-            label="New Account"
+        <SQFormDialogStep label="Quotes">
+          <CardList
+            shouldRenderHeader={false}
+            contentWidth="100%"
+            contentHeight="250px"
+            cardStyle={{minWidth: 'auto'}}
+            isInitiallyExpanded={true}
+            isExpandable={boolean('isExpandable', true)}
+            tabs={tabOptions}
           />
         </SQFormDialogStep>
         <SQFormDialogStep
-          label="Account Info"
-          validationSchema={object({
-            accountID: mixed().when('newAccount', {
-              is: true,
-              then: number()
-                .required()
-                .min(100, 'Since this is a new account we need the number')
-            })
+          label="Dependents"
+          validationSchema={Yup.object({
```

Comment:
<img width="464" alt="Screen Shot 2020-12-16 at 10 06 17 AM" src="https://user-images.githubusercontent.com/61840726/102375548-445d4f00-3f88-11eb-9815-b25b16401cd7.png">
That is the Yup.object.  I can convert it to a normal object and then refactor the places the validation schema is used.

Reply:
Using a plain JS Object isn't working or recommended in the [Formik docs](https://formik.org/docs/guides/validation#validationschema).  I am going to keep it as a Yup.object unless there is some objection or reason not to.

Reply:
ok cool! I think this should be added to our SQForm documentation because I've definitely not been doing it this way, but as you point out the Formik docs are clear on it